pkg/scheduler/server/internal/etcd/config.go

```diff
@@ -78,12 +99,14 @@ func config(opts Options) (*embed.Config, error) {
 	case modes.KubernetesMode:
 		config.Dir = opts.DataDir
 		etcdURL.Host = "0.0.0.0:" + etcdURL.Port()
+		config.AdvertiseClientUrls[0].Scheme = "https"
```

Comment:
Shouldn't this be set in SelfHosted mode too if TLS is enabled?

Reply:
Yes! Thanks
Updated